python/lsst/ts/ess/common/sensor/gill_windsonic.py

```diff
@@ -20,14 +20,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 __all__ = [
-    "WindSensor",
+    "WindsonicSensor",
```

Comment:
I gently suggest DEFAULT_DIRECTION_STR instead of DEFAULT_DIRECTION_VAL for values that sound intrinsically numeric. It's a useful hint to the reader.